src/VkBootstrap.cpp

```diff
@@ -1137,6 +1137,17 @@ PhysicalDeviceSelector::PhysicalDeviceSelector(Instance const& instance) {
 }
 
 detail::Result<PhysicalDevice> PhysicalDeviceSelector::select() const {
+
```

Comment:
I've found that in some compiler warnings report 'empty for loop' around these for loops. I at one point had a `#if !defined(NDEBUG)` surrounding them, but don't know where that code went.